### PR TITLE
Added available state root handler

### LINKED ISSUE #296: Implement StateRootAvailable handler
- Upserts `Anchor` model

### PR DESCRIPTION
**Pull request by 0xsarvesh**
fixes #296  

Added implementation of AvailableStateRootsHandler and tests.

### CODE REVIEW

File: src/common/Utils.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import * as Web3Utils from 'web3-utils';
+
+/**
+ * This class contains general purpose functions.
+ */
+export default class Utils {
+  /**
+   * Returns the checksum address.
+   * @param address Ethereum Address.
+   * @re turns Checksum address.
```
Comment (deepesh-kn) [inline src/common/Utils.ts]:
```suggestion
   * @returns Checksum address.
```

File: src/m1_facilitator/handlers/AvailableStateRootsHandler.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+  /**
+   * Handles the AvailableStateRoots entity records.
+   *
+   * @param records List of AvailableStateRoots.
+   */
+  public async handle(records: any[]): Promise<void> {
+    const contractAddressVsBlockNumberMap = new Map();
+
+    records.forEach((record) => {
```
Comment (deepesh-kn) [inline src/m1_facilitator/handlers/AvailableStateRootsHandler.ts]:
```suggestion
    records.forEach((record): void  => {
```

File: src/m1_facilitator/handlers/AvailableStateRootsHandler.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  public constructor(anchorRepository: AnchorRepository) {
+    this.anchorRepository = anchorRepository;
+  }
+
+  /**
+   * Handles the AvailableStateRoots entity records.
+   *
+   * @param records List of AvailableStateRoots.
+   */
+  public async handle(records: any[]): Promise<void> {
```
Comment (deepesh-kn) [inline src/m1_facilitator/handlers/AvailableStateRootsHandler.ts:42]: Please add
`// eslint-disable-next-line @typescript-eslint/no-explicit-any`

**Review by deepesh-kn — CHANGES_REQUESTED** [review]

File: src/common/Utils.ts (hunk 1 above)
Comment (0xsarvesh) [reply src/common/Utils.ts]: Done

File: src/m1_facilitator/handlers/AvailableStateRootsHandler.ts (hunk 2 above)
Comment (0xsarvesh) [reply src/m1_facilitator/handlers/AvailableStateRootsHandler.ts]: Done

File: src/m1_facilitator/handlers/AvailableStateRootsHandler.ts (hunk 3 above)
Comment (0xsarvesh) [reply src/m1_facilitator/handlers/AvailableStateRootsHandler.ts:42]: Added above L42.

**Review by deepesh-kn — APPROVED** [review]

**Pull request merged**